frontstage/views/secure_messaging/message_get.py

```diff
@@ -48,8 +49,13 @@ def view_conversation(session, thread_id):
             return redirect(url_for('secure_message_bp.view_conversation_list'))
 
     unread_message_count = {'unread_message_count': get_message_count_from_api(session)}
-    survey_name = get_survey(app.config['SURVEY_URL'], app.config['BASIC_AUTH'],
-                             refined_conversation[-1]['survey_id']).get('longName')
+    survey_name = None
+    try:
+        survey_name = get_survey(app.config['SURVEY_URL'], app.config['BASIC_AUTH'],
+                                 refined_conversation[-1]['survey_id']).get('longName')
+    except ApiError as exc:
+        logger.error('Failed to retrieve survey name', status_code=exc.status_code)
```

Comment:
The logging should be info level as the error logging has happened already in the controller.
Additionally, the message here isn't all that useful.  Maybe something like `Failed to get survey name, setting to None`?

Reply:
Done, thanks.